software/slskd.yml

```diff
@@ -0,0 +1,12 @@
+name: "slskd"
+website_url: "https://github.com/slskd/slskd"
+source_code_url: "https://github.com/slskd/slskd"
+description: "A modern client-server application for the Soulseek file sharing network."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - C#
+tags:
+  - File Transfer - Peer-to-peer Filesharing
+depends_3rdparty: true
```

Comment:
```suggestion
```

As far as I know Soulseek is purely peer-to-peer, so there is no dependency on a non-free network service. I don't think this applies.

Reply:
you can't use slskd and nicotine+ without an account on Soulseek and the connection to it.

Reply:
> https://en.wikipedia.org/wiki/Soulseek#Central_server
> Soulseek depends on a pair of central servers [...] these central servers are key to coordinating searches and hosting chat rooms, they do not actually play a part in the transfer of files between users,


I didn't know that, thanks